test/footnote-reference.js

```diff
@@ -8,7 +8,8 @@ test('FootnoteReference', function(t) {
   t.deepEqual(
     to(
       u('footnoteReference', {
-        identifier: 'alpha'
+        identifier: 'alpha',
+        label: 'alpha'
```

Comment:
```suggestion
```

Same as previous (note: the comma after `identifier: 'alpha',` should also be removed)